source/documentation/breaking-changes/bogus-combinators.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Breaking Change: Invalid Combinators'
+introduction: >
+  Sass has historically been very permissive about the use of leading, trailing,
+  and repeated combinators in selectors. These combinators are being deprecated
+  except where they're useful for nesting.
+---
+
+Sass has historically supported three invalid uses of combinators:
+
+- Leading combinators, as in `+ .error {color: red}`.
+
+- Trailing combinators, as in `.error + {color: red}`.
+
+- Repeated combinators, as in `div > > .error {color: red}`.
+
+None of these are valid CSS, and all of them will cause browsers to ignore the
+style rule in question. Supporting them added a substantial amount of complexity
+to Sass's implementation, and made it particularly difficult to fix various bugs
+related to the `@extend` rule. As such, we [made the decision] to remove support
+for these uses.
+
+[made the decision]: https://github.com/sass/sass/issues/3340
+
+**There is one major exception**: leading and trailing combinators may still be
+used for nesting purposes. For example, the following is still very much
+supported:
+
+<!-- prettier-ignore-start -->
+{% codeExample 1 %}
```

Comment:
@jerivas we also moved away from using the same numbers as the source code for the codeExample names and are replacing them with something more descriptive like `{% codeExample 'breaking-changes-trailing-combinators' %}`. There used to be logic to increment those example numbers but we're not doing that anymore so don't need to have numbers here.

Reply:
Thanks, I'll add names to all